config.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+# Safely transfer data
+
+KEY= os.environ.get("SECRET_KEY") or b'\xb0\x9a\xc9\x05\x19\xafv\x88\xfaB\xe5$\x07\x12\x10\x9e'
```

Comment:
What is the use of this KEY? I see that you import it in the __init__.py but don't use it at all.

Reply:
Sorry for the confusion. This commit is just a demo of how to push code to git I showed Phat earlier. I haven't finished fixing the test yet :D